### DISCUSSION TITLE
how can I fix the error  No profiles fo...

### DISCUSSION THREAD

**Discussion by hsumen** [fastlane/fastlane#17807] [General]
No profiles for 'A.Emio' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'A.Emio'. Automatic signing is disabled and unable to generate a profile. To enable automatic signing, pass -allowProvisioningUpdates to xcodebuild. (in target 'Emio' from project 'Emio')

how can I fix the error?

**Comment by rogerluan** [comment]
Hey @hsumen 👋 

From what you posted, looks like you're trying to build and sign a version of an iOS app. To sign it, you need to provide valid provisioning profiles, or allow automatic provisioning to happen. Looks like neither option was provided, so it failed.

We need way more information to be able to help you further 😅 I recommend reading about iOS app signing to help you understand how it's supposed to work 🙏
